sdk/servicebus/Microsoft.Azure.ServiceBus/src/Management/SubscriptionDescription.cs

```diff
@@ -184,7 +184,15 @@ public string ForwardTo
                     return;
                 }
 
-                EntityNameHelper.CheckValidQueueName(value, nameof(ForwardTo));
+                var testValue = value;
+                if (Uri.IsWellFormedUriString(value, UriKind.Absolute))
```

Comment:
You may want to consider combining the test and creation into a single `TryCreate` call:

```csharp
if (Uri.TryCreate(value, UriKind.Absolute, out var uriValue))
{
    value = uriValue.PathAndQuery;
}

value . value.TrimStart('/');
```

Reply:
Thank you for the suggestion @jsquire. I've amended my commit accordingly.

Reply:
@Jesse, the final value still needs to be the FQDN-path. But validation needs to be only on the path, and that's what the current proposed code is doing. 
IMO, the current code looks good to me. 

---
In reply to: [404073801](https://github.com/Azure/azure-sdk-for-net/pull/11059#discussion_r404073801) [](ancestors = 404073801)